holonote/app/tabulator.py

```diff
@@ -14,6 +14,9 @@ class AnnotatorTable(pn.viewable.Viewer):
     annotator = param.Parameter(allow_refs=False)
     tabulator = param.Parameter(allow_refs=False)
     dataframe = param.DataFrame()
+    tabulator_kwargs = param.Dict(
```

Comment:
Sorry for my poor review earlier. I can start to understand why it's not a parameter because it's unclear what happens if you instantiate a tabulator PLUS a tabulator_kwargs.

I think you were right to pop it out of params.

However, instead of using `self.tabulator`, I think you should just pass the kwargs to `self._create_tabulator(**tabulator_kwargs)` and use it in `pn.widgets.Tabulator`.

Reply:
hmm.. I don't understand this statement "what happens if you instantiate a tabulator PLUS a tabulator_kwargs". We are indeed instantiating a tabulator and optionally providing tabulator_kwargs with the `param` approach.